examples/triton_runner/README.md

```diff
@@ -0,0 +1,103 @@
+# Triton Inference Server integration
+
+BentoML now provides support for Triton Inference Server. Triton can be accessed via `bentoml.triton.Runner`:
```

Comment:
Let's add a few bullet points on why users would want to use Triton. A few suggestions.
- User is already using Triton. They can unify model management with other frameworks.
- Performance gaps. While BentoML is constantly improving, under some conditions Triton performs better.
- Adding pre-process logic or multli-model inference graph

Reply:
> What is `examples/triton_runner/BUILD`?

ah I have some bazel rules I tested with locally during the weekend 😄